packages/theme-material-ui/scripts/buildThemes.js

```diff
@@ -0,0 +1,272 @@
+/*
```

Comment:
this commented chunk seems to be leftover copy and paste from `@hig/theme-data`.